runners/direct-java/src/test/java/org/apache/beam/runners/direct/KeyedPValueTrackingVisitorTest.java

```diff
@@ -111,7 +113,13 @@ public void unkeyedInputWithKeyPreserving() {
                                 KvCoder.of(StringUtf8Coder.of(), VarIntCoder.of()))))));
 
     PCollection<KeyedWorkItem<String, KV<String, Integer>>> unkeyed =
-        input.apply(ParDo.of(new ParDoMultiOverrideFactory.ToKeyedWorkItem<String, Integer>()));
+        input
+            .apply(ParDo.of(new ParDoMultiOverrideFactory.ToKeyedWorkItem<String, Integer>()))
+            .setCoder(
```

Comment:
It's not, and this is a `ParDo`, so it has to use the `ParDo#getDefaultOutputCoder` - so we either add `KeyedWorkItem` to the registry (which might be worth doing if possible), but we can't implement one.